docs/docs/tutorial/part-1/index.mdx

```diff
@@ -0,0 +1,299 @@
+---
+title: Create and Deploy Your First Gatsby Site
+---
+
+import { Announcement } from 'gatsby-interface'
+
+## Introduction
+
+Now that you've set up your computer with all the tools you'll need, it's time to get started!
+
+In this part of the Tutorial, you will go through the process of creating your first Gatsby site and deploying it online for everyone to see.
+
+The diagram below shows a high-level view of how all the pieces of this process fit together. (Don't worry if this doesn't make sense yet. You'll learn about each step as you go.)
```

Comment:
Right now, this diagram shows up here and in the Summary section at the end. Are those the right places to convey that information to learners? Would it be better to break it down and talk about the relevant piece in each of the sections?

Reply:
I find it okay to show it at the beginning and as a recap at the end 👍 While I certainly don't want to talk for everyone a "complete" schema at the beginning can help with the following content (at least for me).

What would be important though is that the headings can easily be connected to sections of the diagram

Reply:
I think it does make sense to have it here. I love the diagram (yay more Megan illustrations!) but I wonder if the arrow at 2 should flow the other way: the data goes from GitHub to Gatsby Cloud.

Reply:
@LekoArts good point about the headings. I'll see if I can fix those. Right now they *almost* match, except I added an extra heading for running the site locally.

@ascorbic thanks! :) And yeah good thinking about the arrows. I'll switch the direction on 2 and 3 so that it all flows in one direction.